recipes/integrations.rb

```diff
@@ -30,7 +30,7 @@
 #   'url' => 'http://localhost:22222'
 # }
 node['datadog']['extra_packages'].each do |name, options|
-  package options['name'] do
+  package options['name'].sub('_', '-') do
```

Comment:
couldn't we cover this case with better documentation here and in the `attributes/default.rb` file?

Making the substitution ourselves in this recipe forces us to support this substitution "forever" (changing the behavior later would be a breaking change)